README.md

```diff
@@ -393,6 +393,71 @@ $ curl https://localhost:4430 -k -v
 
 Notice that the issuer in this example is being signed by a training Venafi TPP instance which is not a trusted certificate authority.
 
+### Securing inside cluster communication
```

Comment:
Nit: In the SoW we talk about securing "intra-pod" communication.
Actually, I think that's wrong and that it should be "Securing inter-pod communication" or "Securing Inter-Container Communication".

Reply:
Going for inter container, if we assume the kubernetes knowledge is not very high the term container everyone probably heard of